src/components/pages/main.js

```diff
@@ -1,559 +0,0 @@
-import React, { Component } from 'react'
```

Comment:
Just kind of curious, why was this deleted? It seems like most of it was replaced by vibemap.js? Was this just a previous version of it